utils/common/src/main/kotlin/ProcessCapture.kt

```diff
@@ -17,13 +17,17 @@
  * License-Filename: LICENSE
  */
 
-package org.ossreviewtoolkit.utils.core
+package org.ossreviewtoolkit.utils.common
 
 import java.io.File
 import java.io.IOException
 
 import kotlin.io.path.createTempDirectory
 
+import org.apache.logging.log4j.kotlin.loggerOf
+
+private val log = loggerOf(ProcessCapture::class.java)
```

Comment:
Please use `cachedLoggerOf` (which was recently added to Log4j's Kotlin wrapper).